package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "build:config": "./scripts/create-library-config.js",
-    "build:docs": "npm run build:config && npm run build && npx gulp docs-site && cd docs && bundle install && bundle exec jekyll build --config _config.yml,_config-library.yml && cd ..",
+    "build:docs": "npm run build:config && npm run build && npx gulp docs-site && cd docs && bundle install && bundle exec jekyll build --config _config.yml,_config-prod.yml,_config-library.yml",
+    "build:docs:netlify": "npm run build:config && npm run build && npx gulp docs-site && cd docs && bundle install && bundle exec jekyll build --config _config.yml,_config-library.yml",
```

Comment:
@eboyer had to add some additional changes here for the netlify build - it needed the additional config file excluded or else it was having the same problem as dev mode (css in wrong folder). Can you think of a cleaner way to do this?